templateflow/__about__.py

```diff
@@ -1,68 +1,13 @@
 # emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
 # vi: set ft=python sts=4 ts=4 sw=4 et:
-"""\
-Group inference and reporting of neuroimaging studies require that individual's \
-features are spatially aligned into a common frame where their location can be \
-called standard. \
-To that end, a multiplicity of brain templates with anatomical annotations \
-(i.e., atlases) have been published. \
-However, a centralized resource that allows programmatic access to templates is \
-lacking. \
-TemplateFlow is a modular, version-controlled resource that allows researchers \
-to use templates "off-the-shelf" and share new ones. \
+"""
+Templateflow
 """
 from ._version import get_versions
 __version__ = get_versions()['version']
 del get_versions
 
 __packagename__ = 'templateflow'
```

Comment:
You can drop this one, as well. I restored it in another package because it actually got used. I think in `PipelineDescription`.